cloud/passportauth/example/index.ts

```diff
@@ -1,13 +1,14 @@
-import {BunyanLogger, Logger} from '@raincatcher/logger';
+import { BunyanLogger, Logger } from '@raincatcher/logger';
```

Comment:
I have configured vscode and webstorm with formatter and making sure that our source code is properly formatted using that rules.